web-api/src/config.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright 2022 Xayn AG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, version 3.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+mod cli;
+mod load_config;
+
+use std::{ffi::OsString, fmt::Display, process::exit};
+
+use clap::{CommandFactory, Parser};
+use serde::{de::DeserializeOwned, Serialize};
+
+use self::{cli::Args, load_config::load_config};
+
+/// Loads the config with custom CLI args.
+///
+/// See [`Config.load()`].
```

Comment:
:wrench: this points to a non-existent item, i'm not sure why the ci doesn't complain about this 🤔 
```suggestion
/// See [`load()`].
```